recharges/tasks.py

```diff
@@ -120,10 +180,10 @@ def prep_hotsocket_data(self, recharge_id):
             'as_json': True,
             'token': get_token(),
             'recipient_msisdn': recharge.msisdn,
-            'product_code': 'DATA',  # TODO 37: to dynamically set product code
-            'network_code': 'VOD',  # TODO 37: to dynamically set network code
+            'product_code': recharge.product_code,
+            'network_code': recharge.network_code,
             'denomination': recharge.amount,
-            'reference': recharge.id + settings.HOTSOCKET_REFBASE
+            'reference': recharge_id + settings.HOTSOCKET_REFBASE
```

Comment:
you need cast `settings.HOTSOCKET_REFBASE` as an integer because environment vars are strings